unity/Assets/0Refactored/scenes/experiments/HuygensPrinciple/scripts/InterferencePlate.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Maroon.Physics;
+using Maroon.Physics.HuygensPrinciple;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InterferencePlate : MonoBehaviour
+{
+    [SerializeField]
+    private SlitPlate slitPlate;
+
+    private float waveLength;
```

Comment:
Please follow the naming convention for non serialize fields. e.g. private float  _waveLength;
This is also indicated by the Resharper.